src/braintree/apple/braintreeOnShippingContactSelectedApple.ts

```diff
@@ -29,37 +34,46 @@ export async function braintreeOnShippingContactSelectedApple(event: ApplePayShi
         });
     };
 
-    const response = await callShippingAddressEndpoint(address, false);
-
-    if(response.success){
-        const shippingLinesResponse = await getShippingLines(API_RETRY);
-        const taxResponse = await setTaxes(API_RETRY);
-
-        if(shippingLinesResponse.success && taxResponse.success){
-            const {totalAmountDue} = getTotals();
-            const displayItems = getPaymentRequestDisplayItems().map(
-                ({label, amount}) => ({
-                    label,
-                    amount: getValueByCurrency(amount, currencyCode),
-                }));
-            const {available_shipping_lines: shippingLines} = getShipping();
+    let responseSuccess = false;
+    if (rsaEnabled) {
+        const estimateShippingResponse = await estimateShippingLines(address, API_RETRY);
```

Comment:
This wont set the default shipping line to the order. This only gets the available shipping lines from the platform. Feel free to resolve the thread if this scenario is already covered.

Reply:
Thank you for catch this, added the set default shipping line to the order.